Workspaces/Base/Workspace.cs

```diff
@@ -9,12 +9,13 @@
 
 namespace UnityEditor.Experimental.EditorVR.Workspaces
 {
-	abstract class Workspace : MonoBehaviour, IWorkspace, IInstantiateUI, IUsesStencilRef, IConnectInterfaces, IUsesViewerScale, IControlHaptics, IRayToNode
+	abstract class Workspace : MonoBehaviour, IWorkspace, IInstantiateUI, IUsesStencilRef, IConnectInterfaces,
+		IUsesViewerScale, IControlHaptics, IRayToNode
 	{
 		const float k_MaxFrameSize = 100f; // Because BlendShapes cap at 100, our workspace maxes out at 100m wide
 
 		public static readonly Vector3 DefaultBounds = new Vector3(0.7f, 0.4f, 0.4f);
-		public static readonly Vector3 MinBounds = new Vector3(0.55f, 0.4f, 0.1f);
+		public static readonly Vector3 MinBounds = new Vector3(0.55f, 0f, 0.1f);
```

Comment:
Yes. The frame itself has its own collider, so we do have something for workspaces with 0 height.